docs/Makefile

```diff
@@ -1,50 +1,71 @@
+CURRENT_DIR = $(shell pwd)
+
+.PHONY: clean
 clean:
+	rm -rf generated
```

Comment:
Nit: this can be a one liner

Reply:
Glad you pointed this out.. looking at it I realized it wasn't updated for the "website" directory change 😞 I updated to the right path but kept them as separate commands since they're bigger now.